connectors/sources/generic_database.py

```diff
@@ -17,6 +18,9 @@
 
 WILDCARD = "*"
 
+# Connector will skip the below tables if it gets from the input
+TABLES_TO_SKIP = ["sysutility_ucp_configuration_internal"]
```

Comment:
What's the table? Is it specific to a particular DB?

Reply:
While indexing system table `sysutility_ucp_configuration_internal` of database `msdb`. We are getting [this](https://github.com/elastic/connectors-python/issues/742) Error. So, temporarily we skip this table. If you want we can create one dictionary with the key as the database name and values as a list of tables. So we skip this tables for msdb only.

Reply:
I think we use this `TABLES_TO_SKIP` for other dialects as well. But if you want we can move this to `mssql` data source.

Reply:
Ideally generic sql connector should not have DB-specific logic, only facilitate it.

For example, you can have base method `tables_to_skip` and override it in each individual connector, for example.

Reply:
Since we are planning to skip tables for MS-SQL only, we can move `TABLES_TO_SKIP = {"msdb": ["sysutility_ucp_configuration_internal"]}` to mssql.py and use it with self.tables_to_skip for better code readability. WDYT?

Reply:
Sure, sounds good to me!

Reply:
Done